usermanagerlib/src/main/java/org/pesmypetcare/usermanagerlib/clients/PetManagerClient.java

```diff
@@ -400,15 +395,13 @@ private JSONObject buildPetJson(PetData petData) {
      * @throws IllegalArgumentException When an invalid field value is passed
      */
     private void checkCorrectType(String field, Object newValue) {
-        if ((field.equals(BIRTH) || field.equals(BREED) || field.equals(PATHOLOGIES))
-            && !(newValue instanceof String)) {
+        if ((field.equals(BIRTH) || field.equals(BREED) || (field.equals(PATHOLOGIES)
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Boolean expression complexity is 4 (max allowed is 3).](https://app.codacy.com/gh/Grupo13-PES-Mascotas/PES_My-Pet-Care-Libraries/pullRequest?prid=5403752)